src/events.js

```diff
@@ -221,6 +221,8 @@ function updateSelectedChapter(
       });
       // featured events
       const featuredEvents = eventData.events.filter((it) => it.Featured);
+      const featuredEventsPlaceholder = $("#featured-events-placeholder");
+      featuredEventsPlaceholder.empty();
```

Comment:
nit: i would personally just return all of the elements from the map & join them, so you only need to update the dom once, i.e.
```ts
const featuredEvents = eventData.events.filter((it) => it.Featured);
const featuredEventsElems = featuredEvents.map((featuredEvent) => {
  const localStartDate = new Date(
    featuredEvent.StartTime,
  ).toLocaleDateString(undefined, dateOptions);
  const facebookEventURL = `https://www.facebook.com/${featuredEvent.ID}`;
  const featuredEventElement = ` 
		<div class="full-container full">
   		...
		</div> 
	`;
})
document.querySelector("div.hero-section").replaceChildren(...featuredEventsElems)
```

Reply:
Oh yeah ok will do! I was trying to make as few changes as possible because it didn't seem easy to test. I guess one way is to use ngrok and point the unpublished site to ngrok before publishing. Is that what you do? I'll have to update the README with that too.

Reply:
Nah, I just test it real quick & roll back if something breaks. Lol.

Reply:
Been a while... Easiest thing might just be duplicating the page on webflow, pasting the script contents into a `<script>` tag on the site, and then deleting the test page when done.